migrations/5_test_case_id_type.down.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE test_item
+    ALTER COLUMN test_case_id TYPE INTEGER USING test_case_id::INTEGER;
+ALTER TABLE test_item
+    DROP COLUMN test_case_hash;
```

Comment:
add dropping new indexes

Reply:
In Postgres indexes are dropped automatically with a removed column